tests/unit/src/test/scala/tests/CodeLensesLspSuite.scala

```diff
@@ -188,7 +190,63 @@ class CodeLensesLspSuite extends BaseLspSuite("codeLenses") {
     } yield ()
   }
 
-  def check(name: String, library: String = "")(
+  check("go-to-super-method-lenses")(
+    """package gameofthrones
+      |
+      |abstract class Lannister {
+      |  def payTheirDebts: Boolean
+      |  def trueLannister = payTheirDebts
+      |}
+      |
+      |trait Tywin extends Lannister{
+      |<< payTheirDebts>>
+      |override def payTheirDebts = true
+      |}
+      |
+      |trait Jamie extends Tywin {
+      |<< payTheirDebts>>
+      |override def payTheirDebts = true
+      |}
+      |
+      |trait Tyrion extends Tywin {
+      |<< payTheirDebts>>
+      |override def payTheirDebts = true
+      |}
+      |
+      |trait Cersei extends Tywin {
+      |<< payTheirDebts>>
+      |override def payTheirDebts = false
+      |<< trueLannister>>
+      |override def trueLannister = false
+      |}
+      |
+      |class Joffrey extends Lannister with Jamie with Cersei {
+      |<< payTheirDebts>>
```

Comment:
Is it possible to fix the formatting here?

Reply:
As answered previously, I pasted in previous answer which regex needs to be modified and what rules it must confront to. I could not quickly make it work and and don't want to spend hours working on regex(or parse this manually) that could only improve formatting in tests a little.
If you have can give me regex that will work I will add it and update tests.

Reply:
Ok, let's leave it for now. Sorry for missing the previous answer!